### PR TITLE
Hydraulic Schema Releases

### PR DESCRIPTION
**Pull request by BenTWhite**
Pry isn't quite right yet - but wanted to get in the mix to see what happens

### CODE REVIEW

File: Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/Released/PipeNetworkHydraulicAnalysis.01.00.00.ecschema.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	<ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+	<ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+	<ECSchemaReference name="Analytical" version="01.00.01" alias="anlyt" />
+	<ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU"/>
+	<ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru"/>
+  <ECSchemaReference name="PipeworkPhysical" version="01.00.01" alias="pipphys"/>
+
+  <ECCustomAttributes>
+		<ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+			<SupportedUse>NotForProduction</SupportedUse>
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/Released/PipeNetworkHydraulicAnalysis.01.00.00.ecschema.xml:18]: SupportedUse should be set to "FieldTesting" on the Released version

File: Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/PipeNetworkHydraulicAnalysis.ecschema.xml [hunk 2]
```diff
@@ -10,8 +10,8 @@
 	<ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
 	<ECSchemaReference name="Analytical" version="01.00.01" alias="anlyt" />
 	<ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU"/>
-	<ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru"/>
-  <ECSchemaReference name="PipeworkPhysical" version="01.00.00" alias="pipphys"/>
+	<ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru"/>
+  <ECSchemaReference name="PipeworkPhysical" version="01.00.01" alias="pipphys"/>
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/PipeNetworkHydraulicAnalysis.ecschema.xml:15]: This comment applies to line 6 (github doesn't let me add it up there): The WIP copy of the schema should be incremented when doing a release - e.g. 1.0.1.

File: Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml [hunk 3]
```diff
@@ -10,9 +10,9 @@
 	<ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
 	<ECSchemaReference name="Analytical" version="01.00.01" alias="anlyt" />
 	<ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU"/>
-	<ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru"/>
-	<ECSchemaReference name="NetworkTopology" version="01.00.00" alias="net"/>
-  <ECSchemaReference name="StormSewerPhysical" version="01.00.00" alias="stmswrphys"/>
+	<ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru"/>
+	<ECSchemaReference name="NetworkTopology" version="01.00.01" alias="net"/>
+  <ECSchemaReference name="StormSewerPhysical" version="01.00.01" alias="stmswrphys"/>
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml:16]: This comment applies to line 6 (github doesn't let me add it up there): The WIP copy of the schema should be incremented when doing a release - e.g. 1.0.1.

File: Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/PipeNetworkHydraulicAnalysis.ecschema.xml (hunk 2 above)
Comment (BenTWhite) [reply Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/PipeNetworkHydraulicAnalysis.ecschema.xml:15]: bumped the version to 01.00.01

File: Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml (hunk 3 above)
Comment (BenTWhite) [reply Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml:16]: bumped the version to 01.00.01

File: Domains/3-DisciplineOther/Hydraulics/Released/SewerHydraulicAnalysis.ecschema.01.00.00.xml [hunk 4]
```diff
@@ -0,0 +1,1082 @@
...
+	<ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+	<ECSchemaReference name="Analytical" version="01.00.01" alias="anlyt" />
+	<ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU"/>
+	<ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru"/>
+	<ECSchemaReference name="NetworkTopology" version="01.00.01" alias="net"/>
+  <ECSchemaReference name="StormSewerPhysical" version="01.00.00" alias="stmswrphys"/>
+
+	<ECCustomAttributes>
+		<ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+			<SupportedUse>NotForProduction</SupportedUse>
```
[the remark this answers is not in the document]
Comment (BenTWhite) [reply Domains/3-DisciplineOther/Hydraulics/Released/SewerHydraulicAnalysis.ecschema.01.00.00.xml:19]: done

File: Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/Released/PipeNetworkHydraulicAnalysis.01.00.00.ecschema.xml (hunk 1 above)
Comment (BenTWhite) [reply Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/Released/PipeNetworkHydraulicAnalysis.01.00.00.ecschema.xml:18]: done

**Comment by diegoalexdiaz** [comment]
You got those filenames backwards... 

Your current filenames in Released folders: 
<schema-name>.ecschema.<schema-version>.xml

It should be: 
<schema-name>.<schema-version>.ecschema.xml

**Review by diegoalexdiaz — APPROVED** [review]

**Review by ColinKerr — APPROVED** [review]

**Review by diegoalexdiaz — APPROVED** [review]

**Pull request merged**